serverless/architecture/serverless-serving-architecture.adoc

```diff
@@ -0,0 +1,26 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-serving-architecture"]
+= Knative Serving architecture
+:context: serverless-serving-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Serving supports deploying and managing serverless applications, by providing a set of objects as Kubernetes custom resources (CRs) that define and control the behavior of serverless workloads on an {product-title} cluster.
+
+These CRs are building blocks to address complex use cases, for example:
+
+* Rapidly deploying serverless containers.
+* Automatically scaling pods.
+// TODO: Add other use cases, more advanced tutorials?
+
+== Knative Serving custom resources
```

Comment:
This will need an ID, but you can skip `context` because this is an assembly file.